doc/user_guide/io.rst

```diff
@@ -212,6 +212,8 @@ HyperSpy. The "lazy" column specifies if lazy evaluation is supported.
     +--------------------+--------+--------+--------+
     | EDAX .spc and .spd |    Yes |    No  |    Yes |
     +--------------------+--------+--------+--------+
+    | h5USID .h5         |    Yes |    No  |    No  |
```

Comment:
> * At some point (fairly soon?), support for non linear axis will be added in hyperspy (#1927), it would be nice if you could add some comments to what would be necessary to change when the non linear axis branch is merged.

OK. I added notes in _get_dim_dict()

> * it would be useful to support lazy loading with dask.

Indeed. This is being tested and will be pushed in the next release of pyUSID soon.

> * out of curiosity, why using two underscore `__` instead of only one `_` which seems to be the convention in python.

double underscore prevents those objects from being used outside the module in which they are declared. I typically use these for functions which generally have no purpose outside the module. I typically use single underscore for those internal functions that may be useful for those who know what they are doing. 

> * Limit all lines to a maximum of 79 characters, https://www.python.org/dev/peps/pep-0008/#maximum-line-length

OK. Done.

Reply:
> It seems that it can write too!

Thanks for catching this typo. I fixed it

Reply:
Lazy loading and saving now supported